### PR TITLE
Fixes #21228 - Use the inline messaging warning

### PR DESCRIPTION
**Pull request by rabajaj0509**
This patch makes sure : 
1) There is consistency(refer https://github.com/theforeman/foreman_discovery/pull/362)
2) We follow pattern fly standards :)

### CODE REVIEW

**Comment by ares** [comment]
Thanks @rahulbajaj0509, would you mind attaching before and after screenshots?

**Comment by rabajaj0509** [comment]
before : 
before

After:
after

**Comment by ares** [comment]
@Rohoover any comments for the warning in this modal window? e.g. width vs table width?

**Review by dLobatog — APPROVED** [review]
👍 if @Rohoover approves

**Comment by Rohoover** [comment]
@ares @dLobatog 
Technically it should occupy the full width of the main content area.

**Comment by rabajaj0509** [comment]
looks good? @Rohoover @ares @dLobatog  : 
column_restore

**Comment by Rohoover** [comment]
bitmoji

File: app/views/hosts/_selected_hosts.html.erb [hunk 1]
```diff
@@ -2,43 +2,45 @@
...
+            <%= raw("<th>" + _("Location") + "</th>") if SETTINGS[:locations_enabled]%>
+            <%= raw("<th>" + _("Organization") + "</th>") if SETTINGS[:organizations_enabled]%>
+          </tr>
+        </thead>
+        <tbody>
+          <% associations = [:hostgroup, :environment] %>
+          <% associations << :organization if SETTINGS[:organizations_enabled]  %>
+          <% associations << :location     if SETTINGS[:locations_enabled]  %>
+          <% hosts.includes(*associations).each do |host| %>
+          <tr>
```
Comment (ares) [inline app/views/hosts/_selected_hosts.html.erb:27]: indentation is a bit off, I'll fix that on merge

**Comment by ares** [comment]
Thanks @rahulbajaj0509, merged as 38f8e0b0e94e650f93c7c30aa602f6c81adcca87

File: app/views/hosts/_selected_hosts.html.erb (hunk 1 above)
Comment (rabajaj0509) [reply app/views/hosts/_selected_hosts.html.erb:27]: sorry for that :)

**Pull request closed without merging**